src/package.xml

```diff
@@ -1,20 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Package xmlns="http://soap.sforce.com/2006/04/metadata">
     <types>
-        <members>Country__c-Country Layout</members>
-        <members>District__c-District Layout</members>
-        <members>State__c-State Layout</members>
-        <name>Layout</name>
+        <members>AccountTriggerHelper</members>
+        <name>ApexClass</name>
     </types>
     <types>
-        <members>Admin</members>
-        <name>Profile</name>
+        <members>Trigger_Account</members>
+        <name>ApexTrigger</name>
     </types>
     <types>
-        <members>Country__c</members>
-        <members>District__c</members>
-        <members>State__c</members>
-        <name>SharingRules</name>
+        <members>Account</members>
```

Comment:
Why full account object is retrieved? If need customer fields or page layout or recordtype then retrieve them only.